twake/frontend/src/app/state/recoil/atoms/Router.ts

```diff
@@ -0,0 +1,40 @@
+import { atom, selector, selectorFamily } from 'recoil';
+
+import { ClientStateType } from 'app/services/RouterService';
+
+export const RouterState = atom<ClientStateType | undefined>({
+  key: 'RouterState',
+  default: undefined,
+});
+
+// to be moved in ./selector
```

Comment:
👍🏻

Reply:
@tissarni could you implement this comment before I can merge ?